worker-document-archetype/src/main/resources/archetype-resources/deployment/docker-compose-__workerName__.yml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2015-2017 Hewlett Packard Enterprise Development LP.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version: '3'
+services:
+  ${workerName}:
+    depends_on:
+        - rabbitmq
```

Comment:
Also include the env_file section that the other workers have:

    env_file:
      - ./rabbitmq.env